src/ngraph/runtime/intelgpu/intelgpu_tensor_view.hpp

```diff
@@ -54,6 +54,7 @@ class ngraph::runtime::intelgpu::IntelGPUTensorView : public ngraph::runtime::Te
     /// @param n Number of bytes to read, must be integral number of elements.
     void read(void* p, size_t tensor_offset, size_t n) const override;
 
+    cldnn::memory* get_data_ptr() { return ocl_memory.get(); }
```

Comment:
This should probably be private

Reply:
I did this in the same manner as other backends.